blackduck-artifactory-common/src/main/java/com/synopsys/integration/blackduck/artifactory/modules/inspection/InspectionModule.java

```diff
@@ -126,7 +93,9 @@ public void reinspectFromFailures() {
         reinspectFromFailures(params);
     }
 
-    //////////////////////// Old cron jobs ////////////////////////
+    public void performPolicySeverityUpdate() {
+        inspectionModuleConfig.getRepos().forEach(policySeverityService::performPolicySeverityUpgrade);
```

Comment:
Why does this one stick around?

Reply:
Because he checks the policies on BlackDuck and if the user changed the severity, it will be updated so that blocking works properly. He is a cronJob that wasn't meant to be removed. I accidentally classified it as and "upgrade" when it even says in the name "update".

Reply:
Sounds compelling!